doc/Language/numerics.rakudoc

```diff
@@ -484,6 +484,33 @@ say 42 ∈ +«<42 100 200>; # OUTPUT: «True␤»
 
 Be mindful of these object identity differences and coerce your allomorphs as needed.
 
+=head2 Transforming non-Allomorphs
+
+When we create variables explictly as Str or Numeric types they are not
+shown as Allomorphs:
+
+=begin code
+my $a = "010"; say $a.^name; # OUTPUT: «Str␤»
+my $b = 42;    say $b.^name; # OUTPUT: «Int␤»
+=end code
+
+We can theen explicitly transform them to their Allomorph counterparts:
+
```

Comment:
"then".

Allomorph, Str, and Numeric should be formatted as a link in all cases in text (not code), e.g.:

shown as L«C<Allomorph>|/type/Allomorph»s: